### ISSUE TITLE
[win] SymlinkTest(s) fail on I Builds

### ISSUE THREAD

**Issue by jukzi** [eclipse-platform/eclipse.platform#802]
SymlinkTest(s) keep failing since 2023 10 27
https://download.eclipse.org/eclipse/downloads/drops4/I20231101-1800/testresults/html/org.eclipse.core.tests.resources_ep430I-unit-win32-java17_win32.win32.x86_64_17.html

https://download.eclipse.org/eclipse/downloads/drops4/I20231027-0140/testresults/html/org.eclipse.core.tests.resources_ep430I-unit-win32-java17_win32.win32.x86_64_17.html
```
Multiple Failures (2 failures) org.junit.TestCouldNotBeSkippedException: Test could not be skipped due to other failures java.lang.NullPointerException: Cannot invoke "org.eclipse.core.filesystem.IFileStore.delete(int, org.eclipse.core.runtime.IProgressMonitor)" because "this.localFileBaseStore" is null

org.opentest4j.MultipleFailuresError: Multiple Failures (2 failures)
org.junit.TestCouldNotBeSkippedException: Test could not be skipped due to other failures
java.lang.NullPointerException: Cannot invoke "org.eclipse.core.filesystem.IFileStore.delete(int, org.eclipse.core.runtime.IProgressMonitor)" because "this.localFileBaseStore" is null
Suppressed: org.junit.TestCouldNotBeSkippedException: Test could not be skipped due to other failures
Caused by: org.junit.AssumptionViolatedException: Can't create symbolic links in this platform: win32
at org.junit.Assume.assumeTrue(Assume.java:68)
at org.eclipse.core.tests.filesystem.SymlinkTest.setUp(SymlinkTest.java:106)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
Suppressed: java.lang.NullPointerException: Cannot invoke "org.eclipse.core.filesystem.IFileStore.delete(int, org.eclipse.core.runtime.IProgressMonitor)" because "this.localFileBaseStore" is null
at org.eclipse.core.tests.filesystem.FileSystemTest.tearDown(FileSystemTest.java:101)
at org.eclipse.core.tests.filesystem.SymlinkTest.tearDown(SymlinkTest.java:116)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
```

**Comment by HeikoKlare** [comment]
Duplicate of #783? Will be fixed with #785.

**Comment by jukzi** [comment]
didn't see that, thanks
